src/hardware/mixer.cpp

```diff
@@ -120,7 +118,17 @@ struct mixer_t {
 
 static struct mixer_t mixer = {};
 
-uint8_t MixTemp[MIXER_BUFSIZE] = {};
+alignas(sizeof(void *)) uint8_t MixTemp[MIXER_BUFSIZE] = {};
+
+static inline void MIXER_LockAudioDevice()
```

Comment:
Why inline in an implementation file? It literally only tells the compiler that the definition of the function can be duplicated, hence it's only useful for putting function definitions in header files. It doesn't actually "inline" the function body as some people believe, neither does it give a hint to the optimiser about that.

Reply:
Will do.

There is one existing inline on `Mixer_irq_important()`, and also in general the Core Guidelines: http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f5-if-a-function-is-very-small-and-time-critical-declare-it-inline